examples/solidstart-openai/package.json

```diff
@@ -1,33 +1,28 @@
 {
   "name": "solidstart-openai",
-  "private": true,
+  "type": "module",
   "scripts": {
-    "dev": "solid-start dev",
-    "build": "solid-start build",
-    "start": "solid-start start"
+    "dev": "vinxi dev",
+    "build": "vinxi build",
+    "start": "vinxi start"
```

Comment:
what is vinxi?

Reply:
(I'd prefer to not have an extra framework and stay on the level of solidstart)

Reply:
Vinxi is the new way of building and using solidstart

See here: https://docs.solidjs.com/solid-start/getting-started